src/Components/NavBar/NavBar.css

```diff
@@ -16,6 +16,7 @@
   width: 100px;
   text-align: center;
   border: 1px solid black;
+  border-radius: 5px;
   transition: transform 0.3s ease-in-out;
 }
```

Comment:
Glad to see you updated the nav buttons to match the other button styles! 👏

Reply:
Yes, good attention to detail!